Add peaks.js@0.6.0 w/ npm auto-update

dakshshah96 opened the pull request:
close #11016, cc @bbc

Pull request for issue: #11016 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/bbc/peaks.js/
 * Official website (optional, not the repository): http://waveform.prototyping.bbc.co.uk/
 * NPM package url (optional): https://www.npmjs.com/package/peaks.js
 * License and its reference: LGPL-3.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 51
   - Count of stars: 561
   - Count of forks: 78
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 102
   - Downloads in the last month: 483

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

pvnr0082t commented on ajax/libs/peaks.js/package.json:
```diff
@@ -0,0 +1,31 @@
+  "name": "peaks.js",
+  "version": "0.6.0",
+  "description": "Frontend app for displaying audio waveforms",
+  "homepage": "http://waveform.prototyping.bbc.co.uk/",
```
I can't access this homepage url, can you? If not, let's remove it. Thanks.

dakshshah96 replied:
@pvnr0082t I was able to access the homepage yesterday. Unable to access it right now. I think this is a temporary problem from their side.

dakshshah96 replied:
@pvnr0082t The URL is accessible now 😄

pvnr0082t commented on ajax/libs/peaks.js/package.json:
```diff
@@ -0,0 +1,31 @@
+  "npmName": "peaks.js",
+  "npmFileMap": [
+    {
+      "basePath": "src",
```
We should fetch `peaks.js` and `peaks.js.map` under root of peaks.js npm package.

dakshshah96 commented:
@pvnr0082t Done 👍

pvnr0082t commented:
@dakshshah96 md5sum are still not the same. How did you get the files?
```
037d78049dcb7c0473b5899be5b5d08a  peaks.js
dcbfe5b7aafa9ed98fef9bf5e42aade1  node_modules/peaks.js/peaks.js
```

dakshshah96 commented:
@pvnr0082t I got them from npm by doing `npm install peaks.js` and copied the files directly from my `node_modules/peaks.js` folder. Don't know why they're different 😞

pvnr0082t commented:
@dakshshah96 I think you can try again. :+1:

dakshshah96 commented:
@pvnr0082t Done 👍 md5sum should be fine now.

* Updated files by downloading from [unpkg](https://unpkg.com/peaks.js@0.6.0/)
* Modified commit message to ping repo maintainer instead of organization (@bbc)
